components/automate-ui/e2e/admin.e2e-spec.ts

```diff
@@ -164,57 +165,69 @@ describe('Admin pages', () => {
       });
 
       it('displays tokens table', () => {
-        const headerRows = $$('chef-table chef-thead chef-tr');
-        expect(headerRows.count()).toEqual(1);
-        const tokenRows = $$('chef-table chef-tbody chef-tr');
-        expect(tokenRows.count()).toEqual(2);
+        expectCountElements('chef-table chef-thead chef-tr', 1);
+        expectCountElements('chef-table chef-tbody chef-tr', 2);
       });
 
       it('displays the returned token info', () => {
-        const name = $('chef-table chef-tbody chef-tr:nth-child(1) chef-td:first-child');
-        expect(name.getText()).toBe('This token was generated by the chef-automate CLI tool. ' +
-          'It has admin level access on the entire Automate API.');
-
-        const id = $('chef-table chef-tbody chef-tr:nth-child(1) chef-td:nth-child(2)');
-        expect(id.getText()).toBe('admin-token');
+        //const name = browser.wait(EC.presenceOf($('chef-table chef-tbody chef-tr:nth-child(1) chef-td:first-child')));
```

Comment:
is this for debugging? i'd drop it from here and use it as an example in the browser expectations file